supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -16,17 +16,15 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
 The Conscious Language Group supports the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects with which Red Hat is involved.
 For more information about the Conscious Language Group, see https://github.com/conscious-lang/conscious-lang-docs.
 
-=== blacklist and whitelist
-
 When possible, rewrite documentation to avoid these terms.
 
+=== blacklist and whitelist
 
 .Example: Removing 'blacklist'
```

Comment:
I suggest changing all terms such that they use italics to set off the term rather than single quotes.